### PR TITLE
black fix and drop py2.7

### PR DESCRIPTION
**Pull request by reaperhulk**

### CODE REVIEW

**Comment by alex** [comment]
py27 thingy is now a 404.... time to drop py2 here?

**Comment by reaperhulk** [comment]
Yeah let's drop it.

File: pyproject.toml [hunk 1]
```diff
@@ -9,4 +9,4 @@ build-backend = "setuptools.build_meta"
 
 [tool.black]
 line-length = 79
-target-version = ["py27"]
+target-version = ["py36"]
```
Comment (alex) [inline pyproject.toml:12]: This should be 3.5.

**Review by alex — CHANGES_REQUESTED** [review]
Wheel builder update?

File: pyproject.toml (hunk 1 above)
Comment (reaperhulk) [reply pyproject.toml:12]: I made it 36 on the assumption we'll drop 35 soon, but it's an easy change so sure 😄

**Review by alex — APPROVED** [review]

**Pull request merged**